frontend/src/eventPage/Overview.js

```diff
@@ -0,0 +1,27 @@
+import React, { useContext } from 'react';
+import EventContext from '../contexts/EventContext';
+import { useParams } from 'react-router-dom';
+import styled from 'styled-components/macro';
+
+export default function Overview() {
+  const { events } = useContext(EventContext);
+  const { id } = useParams();
+  const event = events.find((event) => event.id === id);
+  const members = event.members;
```

Comment:
```suggestion
  const {members} = event;
```

;) Since `members` is only used in one place, maybe save the const and directly use `event.members` in line 12?